src/+FeatureCreators/FeatureSetRawRmAmsLRSeparate.m

```diff
@@ -0,0 +1,106 @@
+classdef FeatureSetRawRmAmsLRSeparate < FeatureCreators.Base
+% 
+
+    %% --------------------------------------------------------------------
+    properties (SetAccess = private)
+        freqChannels;       % no. of frequency bins for rate maps
+        amFreqChannels;     % no. of frequency bins for amsFeatures
+        amChannels;         % no. of modulation frequencies
+        wsize;
+        shift;
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Static)
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Access = public)
+        
+        function obj = FeatureSetRawRmAmsLRSeparate( )
+            obj = obj@FeatureCreators.Base( );
+            obj.freqChannels = 16;
+            obj.amFreqChannels = 16;
+            obj.amChannels = 9;
+            obj.wsize = 32e-3;

```

Comment:
in your new fcs, you are using our standard of 20ms and 10ms. Do you want to keep it like this or also adapt?